ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1344,19 +1344,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-private
-- as: gcp-ipi-xpn-private-arm-f14
+- as: gcp-ipi-xpn-private-filestore-csi-arm-f14
```

Comment:
Suggest to sync the same change to `gcp-ipi-xpn-private-amd-f28-destructive` too.